package.json

```
@@ -26,6 +26,7 @@
         "_______ Start Scripts _______": "Here are standalone scripts for running individual applications for development.",
         "suite:dev": "yarn workspace @trezor/suite-web dev",
         "suite:dev:desktop": "yarn workspace @trezor/suite-desktop dev",
+        "mobile:start": "cd suite-native/app && yarn start",
```

I would slightly prefer `native:start` because it just seems to me more consistent, but it's a nitpick.

---

We can use workspace instead of `cd`, same as other scripts:
```suggestion
        "mobile:start": "yarn workspace @trezor/suite-native start",
```

---

Fixed